README.md

```diff
@@ -1,3 +1,34 @@
+# Authorized Greetings - By pobletacio
+
+First add test wallets declared on .env.example (Admin, greeter, unauthorized greeter), then the deploy script itself is gonna airdrop with XLM every accounts set
+
+```bash
+cd soroban-react-dapp/
+docker-compose up -d
+bash run.sh
+cd contracts
+make build
+yarn install
+yarn deploy testnet authorized_greeting
+yarn test testnet
```

Comment:
Nice to add TS tests

Reply:
![image](https://github.com/user-attachments/assets/2ceac1fc-c6fc-4642-8eae-ba56d4a201ba)
nice